### ISSUE TITLE
Automap is not displaying "Items" (as it displays "Enemies" and "Secrets")

### ISSUE THREAD

**Issue by Tormentor667** [Realm667/WolfenDoom#234]
Can this be added as well so it's easier fpr 100% runs to have an overview what has been collected already @AFADoomer ?

**Comment by AFADoomer** [comment]
Already in...  Set [am_showitems](https://zdoom.org/wiki/CVARs:Automap#am_showitems) to true (or set it in the Automap Options menu).

**Comment by Tormentor667** [comment]
Is it off by default then or why wasn’t it visible for me?

**Comment by AFADoomer** [comment]
Yes, it's false by default (since Doom didn't show item count by default).

**Comment by Tormentor667** [comment]
Monsters and secrets are true by default I guess? Would it make sense to somehow make this setting also true by default?

**Comment by AFADoomer** [comment]
It's a GZDoom cvar, so we can't really override it.  I can add a DEFCVARS lump, which should let us override the default value if someone runs BoA as an IWAD/IPK3, at least, though.

**Comment by Tormentor667** [comment]
That would be cool for the standalone in general - for setting the mod in a way how the developers meant it to be (also regarding the forced intermission)

**Comment by AFADoomer** [comment]
Good point...  I'll add a couple of other defaults, too.   Screen wipe style, HUD size (should it default to the fullscreen HUD?), etc.  

The forced intermission CVar always gets reset to false now, so it's less of an issue - the player has to make the conscious choice to use it.

**Comment by Tormentor667** [comment]
Yes, that makes definitely sense, keeping the full screen hud. What about performance settings? Do we go for max visuals?

**Comment by AFADoomer** [comment]
Those we can set in CVARINFO already (they're ours for the mod, not built-in CVars)...  

I think the defaults there are OK, but maybe someone with a less powerful graphics card would disagree.  Right now, we force players to dial things back themselves, rather than starting in the middle and letting players who have the processing power to increase their visuals to do so.  It boils down to: What do we want the "standard" BoA experience to look like?

**Comment by Tormentor667** [comment]
Well, that’s a good question. Generally I’d say we want the mod to be maxed out - make it look the best, but I do understand that some players might try it and say it sucks because it’s not running smooth enough.

I’d personally suggest visuals that show what the mod’s visual features are but not maxed out. People should see first what we created.

**Comment by Ozymandias81** [comment]
Since I am the one with the lower rig here, beside DoomJedi, I suggest to deactivate PostProcess shaders and leave them to players taste.
We could instead force the Bloom effect on, and I guess Heat and Sandstorm should be kept On too.
Blood droplets might need improvements and a proper limit with values since some are too much to handle for everyone.

**Comment by Tormentor667** [comment]
Which postprocessings will be deactivated by default?

**Comment by AFADoomer** [comment]
Right now, none.  The vignette, film grain, and lens flares are the ones I personally disable straight away.  I especially can't stand the lens flares...  That one makes me feel like I'm walking around with dirty glasses on.

**Comment by Ozymandias81** [comment]
Yeah same, neither blur too. Lens needs to be adjusted to use only a couple of values, same for all other PP shaders, so they become more intuitive for players. I'll try something later tonight.

**Comment by Tormentor667** [comment]
I actually like the blur and the lenses flares, as long as they are not that massive :)

**Comment by davidxn** [comment]
For what it's worth, I've left the post-processors on but I feel the film grain is far too high by default, and am more comfortable with it around 0.75 instead of 1.75 (it's a balance, because it's much more obvious in light areas than dark)

**Comment by Tormentor667** [comment]
I agree with the grain factor, this default can be changed to 0.75.

For the items display, is this now activated by default on our new CVARS file @AFADoomer ?

**Comment by AFADoomer** [comment]
Yes.

**Comment by Tormentor667** [comment]
Thanks - aaaand another one closed :)

**Comment by Ozymandias81** [comment]
I will start setting simple values for all these PP shaders when I'll be done with Firebrand sprites :)
